pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -308,28 +310,31 @@ func TestNSMGR_LocalUsecase(t *testing.T) {
 		},
 	}
 
-	conn, err := nsc.Request(ctx, request.Clone())
+	defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
```

Comment:
Please use this at the top of func to prevent goleaks from the code above.

Reply:
Oops, accidentally added this into the PR. Fixed.